### DISCUSSION TITLE
Possible to write a style based on the value of individual cells?

### DISCUSSION THREAD

**Discussion by cparker4486** [openstyles/stylus#1690] [Q&A] [answered]
Considering we generally have no control over the HTML output of the sites for which we write styles, is it possible to write a style based on the content of a <td>?

I want to apply different colors to cells based on the content of the cell. e.g. "Warning" to be orange, "Error" to be red, etc. Best I could find was a `:contains` operator but that doesn't seem to work and the post(s) were from a long time ago.

It would look like this if `:contains` worked:

```CSS
td:contains("Warning") {
    background: orange;
}
```

Is there a way to accomplish it these days?

**Comment by tophf** [comment, marked as answer]
No, `:contains` selector is from jQuery. There is no alternative for CSS. You'll have to make a userscript.
